moralization/data_manager.py

```diff
@@ -284,27 +287,135 @@ def import_data_DocBin(self, input_dir=None, train_file=None, test_file=None):
         )
         return self.spacy_docbin_files
 
-    def lists_to_df(self, sentence_list, label_list):
-        """Convert nested lists of tokens and labels into a pandas dataframe.
+    def docdict_to_lists(self):
+        """Convert the dictionary of doc objects to nested lists."""
 
-        Args:
-            sentence_list (list): A nested list of the tokens (nested by sentence).
-            label_list (list): A nested list of the labels (nested by sentence).
+        # for now work with instantiation
+        tdh = TransformersDataHandler()
+        tdh.get_data_lists(self.doc_dict)
+        tdh.generate_labels(self.doc_dict)
+        self.sentence_list, self.label_list = tdh.structure_labels()
+
+    def lists_to_df(self):
+        """Convert nested lists of tokens and labels into a pandas dataframe.
 
         Returns:
             data_in_frame (dataframe): A list of the train and test files path.
         """
-        data_in_frame = pd.DataFrame(
-            zip(sentence_list, label_list), columns=["Sentences", "Labels"]
+        self.data_in_frame = pd.DataFrame(
+            zip(self.sentence_list, self.label_list), columns=["Sentences", "Labels"]
         )
-        return data_in_frame
 
-    def df_to_dataset(self, data_in_frame, split=True):
-        raw_data_set = datasets.Dataset.from_pandas(data_in_frame)
+    def df_to_dataset(self, data_in_frame: pd.DataFrame = None, split: bool = True):
+        if not data_in_frame:
+            data_in_frame = self.data_in_frame
+        self.raw_data_set = datasets.Dataset.from_pandas(data_in_frame)
         if split:
             # split in train test
-            train_test_set = raw_data_set.train_test_split(test_size=0.1)
-            return train_test_set
-        return raw_data_set
+            self.train_test_set = self.raw_data_set.train_test_split(test_size=0.1)
+
+    def push_dataset_to_hub(
```

Comment:
Would it make sense to use `publish` here to match the ModelManager api (or change the ModelManager `publish`->`push_model_to_hub` to match this one)?